src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,372 @@
+message to the screen. It does however list information not available from the
+Shaka Packager command.
+
+## Change characteristics
+
+### Bit rate
+
+For ffmpeg, I can do this while I'm converting to mp4 or webm.
+ 
+    ffmpeg -i myvideo.mov -b:v 8M myvideo.mp4
+    ffmpeg -i myvideo.mov -vf setsar=1:1 -b:v 8M myvideo.webm
+ 
+For Shaka Packager:
+ 
+    packager \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.mp4 \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4,bandwidth=8000000## Demux (split audio and video)
+
+My preference would be to show file type conversion before diving into demuxing.
+It appears that Shaka Packager cannot do conversion without also demuxing.
+
+**mp4/Shaka Packager**
+ 
+    packager input=myvideo.mp4,stream=video,output=myvideo_video.mp4
+    packager input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+ 
+With Shaka Packager you can combine these.
+ 
+    packager \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4 \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+      
+**webm/Shaka Packager**
+
+    packager \
+      input=myvideo.webm,stream=video,output=myvideo_video.webm \
+      input=myvideo.webm,stream=audio,output=myvideo_audio.webm
+ 
+**mp4/ffmpeg**
+ 
+    ffmpeg -i myvideo.mp4 -vcodec copy -an myvideo_video.mp4
+    ffmpeg -i myvideo.mp4 -acodec copy -vn myvideo_audio.m4a
+    
+**webm/ffmpeg**
+ 
+    ffmpeg -i myvideo.webm -vcodec copy -an myvideo_video.webm
+    ffmpeg -i myvideo.webm -acodec copy -vn myvideo_audio.webm
+
+### File type
+
+Shaka Packager cannot process mov files and hence cannot be used to convert
+files from that format.  
+
+**mov to mp4**
+ 
+    ffmpeg -i myvideo.mov myvideo.mp4
+
+**mov to webm**
+
+When converting a file to webm, ffmpeg doesn't provide the correct aspect
+ration. Fix this with a filter (`-vf setsar=1:1`).
+ 
+    ffmpeg -i myvideo.mov -vf setsar=1:1 myvideo.webm
+
+### File size
+
+    ffmpeg -i myvideo.webm -s 1920x1080 myvideo_1980x1020.webm
+
+### Synchronize audio and video
+
+To ensure that audio and video synchronize during playback insert keyframes.
+ 
+    ffmpeg -i myvideo.mp4 -keyint_min 150 -g 150 -f webm -vf setsar=1:1 out.webm
+ 
+    packager \
+      ??
+
+### Codec
+
+You might have an older file whose codec you want to update.
+
+***mp4/H.264***
+ 
+    ffmpeg -i myvideo.mp4 -c:v libx264 -c:a copy myvideo.mp4
+
+***Audio for an mp4***
+
+    ffmpeg -i myvideo.mp4 -c:v copy -c:a aac myvideo.m4a
+
+***webm/VP9***
+
+    ffmpeg -i myvideo.webm -v:c libvpx-vp9 -v:a copy myvideo.webm
+
+***Audio for a webm***
+
+    ffmpeg -i myvideo.webm -v:c copy -v:a libvorbis myvideo.webm
+    ffmpeg -i myvideo.webm -v:c copy -v:a libopus myvideo.webm
+
+## Demux (split audio and video)
+
+My preference would be to show file type conversion before diving into demuxing.
+It appears that Shaka Packager cannot do conversion without also demuxing.
```

Comment:
"My preference would be" sentence needs to go; either show file type conversion first, or explicitly say "most of you wouldn't care about this, but ShakaPackager requires demuxing in order to convert".